src/app/api/services/workflow/workflow.service.ts

```diff
@@ -12,7 +12,13 @@ import {Workflow} from '../../../shared/models/workflow/workflow.model';
  */
 @Injectable()
 export class WorkflowService {
-    constructor(private coreWebService: CoreWebService) {}
+    private urls: any;
+
+    constructor(private coreWebService: CoreWebService) {
+        this.urls = {
+            getWorkflowSchemes: 'v1/workflow/schemes'
```

Comment:
We do this if the service needs to handle more that one url... in this case you can just put it directly where you define the property in line 15.